frontend/package.json

```diff
@@ -3,16 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@google-cloud/speech": "^4.1.3",
     "@google-cloud/text-to-speech": "^3.1.0",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.0.4",
     "@testing-library/user-event": "^7.2.1",
     "bootstrap": "^4.5.2",
+    "mic-recorder-to-mp3": "^2.2.2",
+    "node-record-lpcm16": "^1.0.1",
     "react": "^16.13.1",
     "react-bootstrap": "^1.3.0",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.2"
+    "react-scripts": "^3.4.3",
+    "sox": "^0.1.0"
```

Comment:
I don't think google-cloud/speech, mic recorder and sox are used, so maybe you can remove unnecessary dependencies?